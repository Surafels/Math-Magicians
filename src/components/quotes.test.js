import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from './quotes';

test('should render', () => {
  const tree = renderer.create(<Quotes />).toJSON();
  expect(tree).toMatchSnapshot();
});
