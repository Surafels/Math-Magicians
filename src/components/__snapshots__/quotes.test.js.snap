// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="quote"
>
  <h2>
    Daily Success Quotes
  </h2>
  <p>
    Loading...
  </p>
</div>
`;
